ethportal-peertest/src/scenarios/find.rs

```diff
@@ -81,16 +81,19 @@ pub async fn test_trace_recursive_find_content(peertest: &Peertest) {
         PossibleHistoryContentValue::ContentPresent(history_content_value)
     );
 
-    let query_origin_node: NodeId = peertest.nodes[0].enr.node_id().into();
-    let node_with_content: NodeId = peertest.bootnode.enr.node_id().into();
+    let query_origin_node: NodeId = peertest.nodes[0].enr.node_id();
+    let node_with_content: NodeId = peertest.bootnode.enr.node_id();
 
     // Test that `origin` is set correctly
     let origin = trace.origin;
-    assert_eq!(origin, query_origin_node);
+    assert_eq!(origin, ethportal_api::NodeId::from(query_origin_node));
```

Comment:
I think if we replace the `NodeId`  type in  `ethportal-api/src/types/query_trace.rs`, we will not need this conversion.